src/pages/index.tsx

```diff
@@ -558,241 +333,296 @@ const HomePage = ({
     },
   ]
 
+  const cardBoxShadow = useToken("colors", "cardBoxShadow")
+
   return (
-    <Page dir={dir}>
+    <Flex flexDirection="column" alignItems="center" dir={dir} width="full">
       <PageMetadata
         title={translateMessageId("page-index-meta-title", intl)}
         description={translateMessageId("page-index-meta-description", intl)}
       />
-      <Hero
+      <Img
+        as={GatsbyImage}
         image={getImage(data.hero)!}
         alt={translateMessageId("page-index-hero-image-alt", intl)}
         loading="eager"
+        width="full"
+        minH="380px"
+        maxH="440px"
+        backgroundSize="cover"
+        background="no-repeat 50px"
+        mb={8}
       />
       <Morpher />
-      <Header>
-        <H1>
-          <Translation id="page-index-title" />
-        </H1>
-        <Description>
-          <Translation id="page-index-description" />
-        </Description>
-        <ButtonLink variant="outline" to="/learn/">
-          <Translation id="page-index-title-button" />
-        </ButtonLink>
-      </Header>
-      <StyledGrayContainer>
-        <StyledContent>
-          <IntroRow>
-            <IntroLeftColumn>
-              <H2>
+      <PageHeader />
+      {/* Getting Started Section */}
+      <GrayContainer>
+        <ContentBox>
+          <Flex
+            alignItems="center"
+            flexDirection={{ base: "column-reverse", md: "row" }}
+            mt={{ md: 4 }}
+            mb={{ md: 12 }}
+          >
+            <Box
+              flex="0 0 50%"
+              maxW={{ lg: "75%" }}
+              p={{ sm: 8, lg: 24 }}
+              boxSize="full"
+            >
+              <SectionHeading fontFamily="inherit" mb={6}>
```

Comment:
I think that all of our section headers should not have the margin top in order to have the content correctly centered. Perhaps we can add it directly on the `SectionHeading` component as its default.
```suggestion
              <SectionHeading fontFamily="inherit" mb={6} mt={0}>
```

Reply:
Applied directly to the component